bugbug/models/backout.py

```diff
@@ -71,7 +89,17 @@ def __init__(self, lemmatization=False):
     def get_labels(self):
         classes = {}
 
+        six_months_ago = datetime.utcnow() - relativedelta(months=6)
+        two_years_and_six_months_ago = six_months_ago - relativedelta(years=2)
+
         for commit_data in repository.get_commits():
+            if self.bug_data:
+                pushdate = dateutil.parser.parse(commit_data["pushdate"])
+                if not (
+                    pushdate >= two_years_and_six_months_ago
+                    and pushdate <= six_months_ago
```

Comment:
We don't need this side here, just pushdate >= two_years_and_six_months_ago.
Also add a comment saying that we restrict the pushdata when we use the bug data because we don't have bugs for all commits and that we need to keep the dates here and in bug_retriever in sync.